docs/whatsnew/2.1.rst

```diff
@@ -0,0 +1,215 @@
+    >>> from sunpy.coordinates import get_body_heliographic_stonyhurst
+    >>> mars = SkyCoord(get_body_heliographic_stonyhurst('mars', '2021-01-01',
+    ...                                                  include_velocity=True))
+    >>> mars
+    <SkyCoord( HeliographicStonyhurst: obstime=2021-01-01T00:00:00.000): (lon, lat, radius) in (deg, deg, AU)
+        (-34.46752135, 1.77496469, 1.50936573)
+     (d_lon, d_lat, d_radius) in (arcsec / s, arcsec / s, km / s)
+        (-0.00048971, 0.00060976, 19.54950062)>
+    >>> mars.velocity.norm()
+    <Quantity 19.56823076 km / s>
+
+However, `~sunpy.coordinates.frames.HeliographicStonyhurst` is a non-inertial frame that rotates over time.
+By transforming this coordinate to `~sunpy.coordinates.frames.HeliocentricInertial`, we can see that Mars's actual velocity is larger::
+
+    >>> mars.heliocentricinertial
+    <SkyCoord (HeliocentricInertial: obstime=2021-01-01T00:00:00.000): (lon, lat, distance) in (deg, deg, AU)
+        (-9.91128592, 1.77496469, 1.50936573)
+     (d_lon, d_lat, d_distance) in (arcsec / s, arcsec / s, km / s)
+        (0.04174239, 0.00060976, 19.54950058)>
+    >>> mars.heliocentricinertial.velocity.norm()
+    <Quantity 49.68592218 km / s>
+
+See :ref:`sunpy-coordinates-velocities` for more information.
+
+.. _whatsnew-2.1-reprojecting:
+
+Alternatives for reprojecting a Helioprojective map
+===================================================
+The typical observation in `~sunpy.coordinates.frames.Helioprojective` coordinates does not contain full 3D information for the sources of emission, so an assumption needs to be made when transforming such coordinates to other coordinate frames.
+By default, SunPy assumes that the emission is coming from the surface of the Sun, which enables reprojections such as in the example :ref:`sphx_glr_generated_gallery_map_transformations_reprojection_different_observers.py`.
+However, this assumption is not appropriate for some observations, e.g., from coronagraphs.
+
+There is now a context manager (:meth:`~sunpy.coordinates.frames.Helioprojective.assume_spherical_screen`) to override the default assumption such that any 2D coordinates are interpreted as being on the inside of a large spherical screen.
+See the following example for how this context manager enables alternative reprojections.
+
+.. minigallery:: sunpy.coordinates.Helioprojective.assume_spherical_screen
+
+.. _whatsnew-2.1-performance:
+
+Performance improvements
+========================
+Several functions in `sunpy.map.maputils` have been significantly sped up with improved algorithms.
+
+In addition, `sunpy.map.Map.wcs` is now cached when the map metadata remains unchanged, significantly improving performance in applications which make mutiple requests for the map WCS (e.g. plotting), and reducing the number of repeated warnings thrown when metadata is missing.
+
+.. _whatsnew-2.1-goes1617:
+
+Support for GOES-16 and GOES-17 X-Ray Sensor (XRS) data
```

Comment:
This should be moved much much higher in the What's New, possibly even to be the first entry.